onnxruntime/core/providers/shared_library/provider_bridge_provider.cc

```diff
@@ -45,15 +45,9 @@ struct OnUnload {
 }  // namespace onnxruntime
 
 // Override default new/delete so that we match the host's allocator
-void* operator new(size_t n) { return g_host->HeapAllocate(n); }
-void operator delete(void* p) { return g_host->HeapFree(p); }
-void operator delete(void* p, size_t /*size*/) { return g_host->HeapFree(p); }
-
-namespace onnx {
-std::unique_ptr<ONNX_NAMESPACE::Provider_AttributeProto> Provider_AttributeProto::Create() {
-  return g_host->AttributeProto_Create();
-}
-}  // namespace onnx
+void* operator new(size_t n) { return onnxruntime::g_host->HeapAllocate(n); }
```

Comment:
>HeapAllocate [](start = 59, length = 12)

Let's rename them. This looks like Windows API too much. MemoryAlloc/MemoryFree

Reply:
What do you suggest? They are functions to allocate & free from the host heap.

Reply:
(plus their inside an interface so they don't pollute the global namespace anyways)